src/nixpkgs.rs

```diff
@@ -18,12 +18,16 @@ use crate::package::{PathOrigin, StorePath};
 /// The `nixpkgs` argument can either be a path to a nixpkgs checkout or another expression
 /// accepted by `nix-env -f`, such as `<nixpkgs>` or `http://example.org/nixpkgs.tar.bz`.
 ///
+/// If system is `Some(attr)`, nix-env is called with the `--argstr system attr` argument so that
+/// the specified platform `attr` would be used instead of the default host system platform.
```

Comment:
This is super nitpicky, but should `attr` be changed to `platform` here since it isn't specifying a nixpkgs attr path (like the help doc below this one is)?

Reply:
Oh no your nitpicking is completely vaild. I force pushed ffce80ca4e809076d4b7c066186ce755f5973235 and thanks!